### DISCUSSION TITLE
No data since March 17

### DISCUSSION THREAD

**Discussion by mortgad** [dcaribou/transfermarkt-datasets#289] [Q&A] [answered]
Hi David,

It seems like multiple files lack data after the 17th of March.
https://www.kaggle.com/datasets/davidcariboo/player-scores/data

I see the pattern in:
- appearances.csv
- games.csv
- game_events.csv

Hope you can assist
Morten

**Comment by dcaribou** [comment, marked as answer]
Hey @mortgad ,
There's seems to be a problem with linueps checks that blocked the pipeline. Let me unblock it until I have more time to look into it.

**Reply by dcaribou** [reply]
Should be ok now @mortgad

**Reply by mortgad** [reply]
Thanks! I see it now. 
The fixtures from the midweek are not included though. Is it a matter of scheduling?

**Reply by dcaribou** [reply]
@mortgad - it was blocked due to the same issue. I just unblocked it, it should be updated now.

**Reply by mortgad** [reply]
Awesome.

Now I get an API error from Kaggle when trying to download your dataset...

<img width="882" alt="image" src="https://github.com/dcaribou/transfermarkt-datasets/assets/48317690/0f105f30-598d-4a7a-ad30-e9f6e39786c4">

This code usually works

**Reply by mortgad** [reply]
Copying the API command from Kaggle gives me this CLI command: kaggle datasets download -d davidcariboo/player-scores
When I execute it in terminal I get this:

<img width="640" alt="image" src="https://github.com/dcaribou/transfermarkt-datasets/assets/48317690/0897737b-c042-4b73-bc77-292daf230583">

**Reply by dcaribou** [reply]
That command did work for me.
It might have been a temporary thing in Kaggle services. They sent this announcement that they'd do changes in Kaggle Datasets today...

<img width="1008" alt="image" src="https://github.com/dcaribou/transfermarkt-datasets/assets/16071835/0575ff3b-6dc2-4aee-996c-fd9c8b35d8f5">
